API.yaml

```diff
@@ -183,11 +185,100 @@ paths:
               description: "Directives for caching the response. TEK Chunks are cached for 15 days."
         404:
           description: "TEK Chunk not found."
+  /analytics/apple/operational-info:
+    post:
+      tags:
+      - "Analytics"
+      summary: "Save the operational data in compliance with the monthly policy, avoiding multiple from the same user in a small timeframe, authorized with the provided analytics token."
+      consumes:
+      - "application/json; charset=utf-8"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "header"
+        name: "Immuni-Dummy-Data"
+        type: "integer"
+        required: true
+        minimum: 0
+        maximum: 1
+      - in: "body"
+        name: "body"
+        description: "A JSON-formatted dictionary containing the operational data."
+        required: true
+        schema:
+          $ref: "#/definitions/OperationalInfo"
+      responses:
+        204:
+          description: "Operation completed successfully."
+        400:
+          description: "Invalid data. An APIError is returned."
+          schema:
+            $ref: "#/definitions/APIError"
+      security:
+        - analytics_bearer: []
+  /analytics/google/operational-info:
+    post:
+      tags:
+      - "Analytics"
+      summary: "Save the operational data sent by the Android device, authorized with the provided SafetyNet hardware attestation token."
+      consumes:
+      - "application/json; charset=utf-8"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "header"
+        name: "Immuni-Dummy-Data"
+        type: "integer"
+        required: true
+        minimum: 0
+        maximum: 1
+      - in: "body"
+        name: "body"
+        description: "A JSON-formatted dictionary containing the operational data."
+        required: true
+        schema:
+          $ref: "#/definitions/GoogleOperationalInfo"
+      responses:
+        204:
+          description: "Operation completed successfully."
+        400:
+          description: "Invalid data. An APIError is returned."
+          schema:
+            $ref: "#/definitions/APIError"
+  /analytics/apple/token:
+    post:
+      tags:
+      - "Analytics"
+      summary: "Authorize the provided analytics token with the device authenticity SDK offered by the device operating system."
+      consumes:
+      - "application/json; charset=utf-8"
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "A JSON-formatted dictionary containing the tokens."
+        required: true
+        schema:
+          $ref: "#/definitions/AnalyticsToken"
+      responses:
+        201:
+          description: "Token is authorized."
```

Comment:
Is there a payload body here?

Reply:
no response body

Reply:
In this case, I'd add that info in the description.